lib/apiservers/engine/backends/cache/image_cache.go

```diff
@@ -181,7 +180,19 @@ func (ic *ICache) getImageByNamed(named reference.Named) (*metadata.ImageConfig,
 		}
 	}
 
-	return copyImageConfig(config), nil
+	return copyImageConfig(config)
+}
+
+// Don't assume the image id in image has "sha256:<id> as format.  We store it in
+// this format to make it easier to lookup by digest
+func prefixImageID(imageID string) string {
+	// Prefix was just "sha" which seems weird to be. Following general logic,
+	// I have update it to "sha256:"
```

Comment:
Docker allows for different algorithms (see [here](https://github.com/docker/docker/blob/b248de7e332b6e67b08a8981f68060e6ae629ccf/vendor/src/github.com/docker/distribution/digest/digester.go#L16)), but since we're pinned to 1.11.2, we know we will never need another one so this is fine.

Just FYI!